packs/data/pathfinder-bestiary-3.db/green-man.json

```
@@ -2858,13 +3012,13 @@
                 "variants": {
                     "0": {
                         "label": "+44 vs. plants",
-                        "options": "against-plants"
+                        "options": "target:tag:plant"
```

```suggestion
                        "options": "target:trait:plant"
```
Plant is an actual trait.